packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -49,10 +50,13 @@ export function useSliderThumb(
   let {addGlobalListener, removeGlobalListener} = useGlobalListeners();
 
   let labelId = sliderIds.get(state);
+  let id = getSliderThumbId(state, index);
+  let thumbId = `${id}-thumb`;
   const {labelProps, fieldProps} = useLabel({
     ...opts,
-    id: getSliderThumbId(state, index),
-    'aria-labelledby': `${labelId} ${opts['aria-labelledby'] ?? ''}`.trim()
+    id,
+    'aria-label': undefined,
```

Comment:
Add a comment to explain what's going on here:

```suggestion
    // Override due to a Chrome bug where aria-labelledby cannot be a self-reference.
    // Instead, we put the label on the thumb element and point to it with aria-labelledby.
    // See https://bugs.chromium.org/p/chromium/issues/detail?id=1159567
    'aria-label': undefined,
```